### ISSUE TITLE
A11y_Azure Data Studio extension 'Azure SQL Migration'_Databases for assessment_Status messages: Narrator/NVDA is not announcing no. of databases(1/103 databases selected) selected message displayed on screen after selecting databases

### ISSUE THREAD

**Issue by NiharikaTipirishetty** [microsoft/azuredatastudio#19845]
"[Check out Accessibility Insights! ](https://accessibilityinsights.io/)- Identify accessibility bugs before check-in and make bug fixing faster and easier.”

### Github Tags:
#A11yMAS; #A11yTCS; #A11ySev2; #A11yWCAG2.1; #Benchmark; #SH-AzureSQLMigration-Win32-Jun2022; #DesktopApp; #Win32; #SQL Migration Control Plane; #WCAG4.1.3; #Status Messages; #Narrator; #NVDA;

### Environment and OS details:
Azure Data Studio Version 1.37.0(User Setup)
Windows Version 21H2 (OS Build 22000.675)

### Note:
This issue is also observed in "Assessment and results recommendations" step after selecting databases in "View/select" dialog

### Repro Steps:

1. Install Azure data studio app and install Azure SQL migration extension
2. Connect to SQL server, tab till respective server, right click and select manage
3. Tab till Azure SQL Migration in sitemap and press enter key
4. Tab till migrate to azure sql and press enter
5. Navigate to databases grid in "Databases for assessment" step
6. Select few databases and verify whether screen reader is announcing 1/103 databases selected status message or not.

### Actual Result:
Narrator/NVDA is not announcing no. of databases(1/103 databases selected) selected message displayed on screen after selecting databases

### Expected Result:
Narrator/NVDA should announce no. of databases(1/103 databases selected) selected message displayed on screen after selecting databases

### Attachment:
![NVDA_databases grid](https://user-images.githubusercontent.com/83570638/176163296-51fdec37-a1ff-4067-ac1e-57d704033b2a.png)
![NVDA_viewselect](https://user-images.githubusercontent.com/83570638/176163339-57917d20-e558-4207-a0dc-45f37329ceb7.png)
![Narrator_databases_](https://user-images.githubusercontent.com/83570638/176164099-fa4eeb4e-a8d2-4992-a5f8-6dee378bb8e0.gif)

**Comment by VijayaBharathyA** [comment]
#Closed;
Verified the issue in below Environment:
Issue is Fixed
Version: 1.41.1 (user setup)

![Status message_Narrator_1](https://user-images.githubusercontent.com/100189407/218033632-a4a1854a-fbc0-4f8d-95d7-4f3b27ee3e32.png)
![status Message_NVDA](https://user-images.githubusercontent.com/100189407/218033637-58eca45c-b343-45e5-9f9e-9d625823d939.png)
![Narrator_2](https://user-images.githubusercontent.com/100189407/218043430-de6d32ba-346d-4156-9460-78ae951e2df5.png)
![Screenshot (24)](https://user-images.githubusercontent.com/100189407/218043779-567a9b28-e32f-4c2f-a936-93b543b9f4b0.png)
